src/gps/gpsTask.c

```diff
@@ -64,6 +67,8 @@ void GPSTask(void *pvParameters)
                         if (result == GPS_MSG_SUCCESS) {
                                 const GpsSnapshot snap = getGpsSnapshot();
                                 lapstats_processUpdate(&snap);
+                                auto_logger_gps_sample_cb(&snap.sample);
```

Comment:
I'd imagine in the future when triggering on other samples we'd move this somewhere central. loggerTask?